plugins/modules/zos_copy.py

```diff
@@ -121,9 +122,16 @@
         type: str
   force:
     description:
-      - If set to C(true), the remote file or data set will be overwritten.
-      - If set to C(false), the file or data set will only be copied if the destination
-        does not exist.
+      - If set to C(true) and the remote file or data set C(dest) is empty,
```

Comment:
Well, I just ran a playbook to copy into an empty data set without using `force` and it managed to copy just fine, so it will be necessary to update that bit of the documentation.